spec.html

```diff
@@ -6241,20 +6254,36 @@
     </emu-clause>
   </emu-clause>
 
+  <emu-clause id="sec-continents">
+    <h1>Continents</h1>
+
+    <p>A <dfn id="continent">continent</dfn> is an independent unit of ECMAScript computation, and is the largest unit that this specification deals with. Everything within this specification is assumed to occur within a single continent.</p>
+
+    <p>The host environment must create a continent prior to the execution of any Jobs or the evaluation of any ECMAScript code.</p>
+
+    <p>A continent may contain multiple realms, as defined in <emu-xref href="#sec-code-realms"></emu-xref>. Each realm belongs to exactly one continent. Depending on the host environment, code executed within a given continent's realms may or may not be able to reach objects from other realms in that continent. However, code executed in one continent must not be able to directly reference or share objects from another continent.</p>
```

Comment:
This paragraph creates a normative requirement that continents can't share or reference with each-other. Doesn't this invalidate the opening sentence that continents are the largest unit the specification deals with (since it is now prescribing how the planet's continents may communicate)? Also the terms "share" and "reference", are they defined rigorously already? In general, it seems ok to omit this paragraph and leave inter-continent communication up to host discretion.